src/main/java/aquarium/shop/order/OrderController.java

```diff
@@ -0,0 +1,53 @@
+package aquarium.shop.order;
+
+
+import aquarium.shop.product.Product;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.web.bind.annotation.PatchMapping;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RestController;
+
+import javax.annotation.PostConstruct;
+import java.math.BigDecimal;
+import java.util.Collections;
+import java.util.HashMap;
+import java.util.List;
+
+@RestController
+public class OrderController {
+
+    @Autowired
+    private OrderRepository orderRepository;
+
+    @PostConstruct
+    public List<OrderDetails> getAllOrders() {
```

Comment:
Please change method name - it's not "getAllOrders". It can be also just void return type.
We should also move this test to different class, it's not controller responsibility to add some verification data.
Did you test it? Was DB populated after startup?

Reply:
Yes, it working fine.